### PR TITLE
Add new html files

### LINKED ISSUE #1: Add html document
Description

### PR DESCRIPTION
**Pull request by Azhng**
Fix #1
See files for change

### CODE REVIEW

File: new.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+	Stuff
```
Comment (9at8) [inline new.html]: :-1:

Comment (9at8) [reply new.html]: jk :+1: lgtm

**Review by 9at8 — DISMISSED** [review]
wut

**Comment by Azhng** [comment]
Explanation about the code blah

**Review by 9at8 — APPROVED** [review]
:sparkles:

**Pull request merged**